app/assets/javascripts/components/utils.js.coffee

```diff
@@ -132,3 +132,9 @@ class @Utils
       .fail (xhr, status, error) ->
         alert('Error: ' + error)
         callback()
+
+  @select2TagClickHandler: (e, elem) ->
+    if e.which == 1
```

Comment:
I like the links if I feel like there's something wrong in a pipeline. I can start at the top (or bottom) in edit mode, and click through to each step and review what's happening until I find the undesirable behavior, fix it, done. 

To @cantino's point about a confirm, feels like there should be a confirm dialog any time you navigate away from unsaved agent changes. But perhaps that's a separate issue?